docs/pages/docs/environments/server-client-components.mdx

```diff
@@ -347,11 +347,6 @@ export default function LocaleLayout({children, params: {locale}}) {
 }
 ```
 
-<Callout type="warning">
-  Note that this is a tradeoff in regard to performance (see the bullet points
-  at the top of this page).
-</Callout>
```

Comment:
Too dramatic IMO, we already told the user everything relevant at the top of the page :).

Reply:
Definitely agree, the wording scared me off a bit when I read this for the first time 😁